components/src/components/Common/Evaluator.ts

```diff
@@ -0,0 +1,7 @@
+import _ from 'lodash';
```

Comment:
We're not using lodash in this file at all.

Reply:
removed in latest commit